### PR TITLE
Release v3.5.6

### PR DESCRIPTION
**Pull request by svetasmirnova**
- [ ] The contributed code is licensed under GPL v2.0
- [ ] Contributor Licence Agreement (CLA) is signed
- [ ] util/update-modules has been ran
- [ ] Documentation updated
- [ ] Test suite update

### CODE REVIEW

File: Changelog [hunk 1]
```diff
@@ -1,5 +1,32 @@
 Changelog for Percona Toolkit
 
+v3.5.6 released 2023-12-
```
Comment (alina-derkach-oaza) [inline Changelog]:
```suggestion
v3.5.6 released 2023-12-21
```

File: docs/release_notes.rst [hunk 2]
```diff
@@ -1,6 +1,45 @@
 Percona Toolkit
 ***************
 
+v3.5.6 released 2023-12-
```
Comment (alina-derkach-oaza) [inline docs/release_notes.rst]:
```suggestion
v3.5.6 released 2023-12-21
```

**Review by alina-derkach-oaza — APPROVED** [review]

File: docs/percona-toolkit.pod [hunk 3]
```diff
@@ -575,6 +575,6 @@ Place, Suite 330, Boston, MA  02111-1307  USA.
 
 =head1 VERSION
 
-Percona Toolkit v3.5.6 released 2023-12-18
+Percona Toolkit v3.5.6 released 2023-12-20
```
Comment (eleo007) [inline docs/percona-toolkit.pod:578]: Wrong date

**Review by EvgeniyPatlan — APPROVED** [review]

File: config/scripts/pt_builder.sh [hunk 4]
```diff
@@ -233,7 +233,7 @@ install_deps() {
               sleep 1
           done
       fi
-      if [ $DEBIAN_VERSION = bionic -o $DEBIAN_VERSION = focal -o $DEBIAN_VERSION = bullseye -o $DEBIAN_VERSION = buster -o $DEBIAN_VERSION = xenial -o $DEBIAN_VERSION = jammy ]; then
+      if [ $DEBIAN_VERSION = bionic -o $DEBIAN_VERSION = focal -o $DEBIAN_VERSION = bullseye -o $DEBIAN_VERSION = buster -o $DEBIAN_VERSION = bookworm -o $DEBIAN_VERSION = jammy -o $DEBIAN_VERSION = xenial ]; then
```
Comment (eleo007) [inline config/scripts/pt_builder.sh:236]: Do we keep bionic here?

Comment (svetasmirnova) [reply config/scripts/pt_builder.sh:236]: Hm... It is a good question.

File: docs/percona-toolkit.pod (hunk 3 above)
Comment (svetasmirnova) [reply docs/percona-toolkit.pod:578]:
```suggestion
Percona Toolkit v3.5.6 released 2023-12-21
```

**Pull request merged**